src/style.css

```diff
@@ -1,3 +1,76 @@
-body {
+:root {
+  --blue: #00acdc;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  padding: 0 50px;
+  margin: 0;
+  justify-content: flex-end;
+}
+
+.nav {
+  display: flex;
+  justify-content: flex-end;
+  padding-left: 0;
+}
+
+.header .nav__item {
+  display: flex;
+  margin-left: 20px;
+  height: 60px;
+  line-height: 60px;
```

Comment:
set height only for nav__link
![image](https://github.com/mate-academy/layout_moyo-header/assets/55200530/e3980e55-8e9d-49c3-850f-ab3a3f9dbe5a)